app/uk/gov/hmrc/accessibilitystatementfrontend/views/AccessibilityStatementTemplate.scala.html

```diff
@@ -16,176 +16,48 @@
         implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-@serviceAbsoluteUrl = {https://@statement.serviceDomain@statement.serviceUrl}
-@serviceOrApp = {@serviceOrAppHelper(statement.mobilePlatform)}
-<h1 class="govuk-heading-xl">@{messages(if (statement.mobilePlatform.isEmpty) "introduction.heading" else "introduction.app-heading", statement.serviceName)}</h1>
-<p class="govuk-body-l">@{messages("introduction.lede", serviceOrApp)}</p>
-@if(statement.mobilePlatform.isEmpty) {
-    <p class="govuk-body">
-        @{messages("introduction.scope.part1")}
-        <a class="govuk-link" href="https://www.gov.uk/help/accessibility">@{messages("introduction.scope.part2")}</a>@{messages("general.fullstop")}
-    </p>
-}
-<p class="govuk-body">@{messages(if (statement.mobilePlatform.isEmpty) "introduction.service-name" else "introduction.app-service-name", statement.serviceName, serviceAbsoluteUrl)}</p>
 
-<h2 class="govuk-heading-l">@{messages("using-service.heading", serviceOrApp)}</h2>
-<p class="govuk-body">@{statement.serviceDescription}</p>
-<p class="govuk-body">
-    @{messages("using-service.this-service-or-app-is-run", serviceOrApp)}
-    @if(statement.mobilePlatform.isEmpty) {
-      @messages("using-service.service.this-means")
-    } else {
-        @messages("using-service.app.this-means")
-        <a class="govuk-link"
-        href="@messages(s"using-service.app.accessibility-features.link.${statement.mobilePlatform.getOrElse("")}")"
-        >@messages("using-service.app.accessibility-features")</a>
-        @messages("using-service.app.on-your-device")
-    }
-</p>
-<ul class="govuk-list govuk-list--bullet">
-@statement.mobilePlatform match {
-  case Some(Android) => {
-    <li>@{messages("using-service.android.bullet1")}</li>
-    <li>@{messages("using-service.android.bullet2")}</li>
-    <li>@{messages("using-service.android.bullet3")}</li>
-    <li>@{messages("using-service.android.bullet4")}</li>
-    <li>@{messages("using-service.android.bullet5")}</li>
-  }
-  case Some(Ios) => {
-    <li>@{messages("using-service.ios.bullet1")}</li>
-    <li>@{messages("using-service.ios.bullet2")}</li>
-    <li>@{messages("using-service.ios.bullet3")}</li>
-    <li>@{messages("using-service.ios.bullet4")}</li>
-    <li>@{messages("using-service.ios.bullet5")}</li>
-  }
-  case None => {
-      <li>@{messages("using-service.bullet1")}</li>
-      <li>@{messages("using-service.bullet2")}</li>
-      <li>@{messages("using-service.bullet3")}</li>
-      <li>@{messages("using-service.bullet4")}</li>
-      <li>@{messages("using-service.bullet5")}</li>
-  }
-}
-</ul>
-<p class="govuk-body">@{messages("using-service.simple-text", serviceOrApp)}</p>
-<p class="govuk-body">
-    <a class="govuk-link" href="https://mcmw.abilitynet.org.uk">@{messages("using-service.making-device-easier.part1")}</a>
-    @{messages("using-service.making-device-easier.part2")}
-</p>
+@defining(statement) { implicit statement =>
```

Comment:
Not totally sure on this pattern of making the statement `implicit`, I'd prefer to pass through only the parameters needed and make those explicit, but happy to discuss.